features/backoffice_deregister_exemption.feature

```diff
@@ -0,0 +1,21 @@
+@backoffice
+Feature: Back office user deregisters a waste exemption activity from a registration
+  As a back office user
+  I want to deregister a waste exemption
+  So that the exemption is deregistered for a valid registration with mutlipe exemptions
```

Comment:
There's a spelling error (mutlipe = multiple) but this is more a case of could we give a better reason, and one that is more generic. We shouldn't be referencing multiple exemptions and instead just have separate scenarios which covers registrations with single and multiple exemptions (these can come later).